src/pages/Inventory.page.js

```diff
@@ -12,4 +12,49 @@ export class InventoryPage extends BaseSwagLabPage {
     async addItemToCartById(id) {
         await this.addItemToCartButton.nth(id).click();
     }
+
+    // Generate and return random number  generateRandomNumber(0, 6)  from 0 to 5 include.
+    static generateRandomNumber(min, max){
```

Comment:
We would better have this in some helpers file as this is not related to the page.